docs/content/connectors/oceanbase-cdc.md

```diff
@@ -0,0 +1,492 @@
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>connector</td>
+                <td>required</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>Specify what connector to use, here should be <code>'oceanbase-cdc'</code>.</td>
+            </tr>
+            <tr>
+                <td>scan.startup.mode</td>
+                <td>required</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>Specify the startup mode for OceanBase CDC consumer, valid enumerations are
+                    <code>'initial'</code>,<code>'latest'</code> or <code>'timestamp'</code>.
+                </td>
+            </tr>
+            <tr>
+                <td>scan.startup.timestamp</td>
+                <td>optional</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>Long</td>
+                <td>Timestamp in seconds of the start point, only used for <code>'timestamp'</code> startup mode.</td>
+            </tr>
+            <tr>
+                <td>username</td>
+                <td>required</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>Username to be used when connecting to OceanBase.</td>
+            </tr>
+            <tr>
+                <td>password</td>
+                <td>required</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>Password to be used when connecting to OceanBase.</td>
+            </tr>
+            <tr>
+                <td>tenant-name</td>
+                <td>required</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>Tenant name of OceanBase to monitor.</td>
+            </tr>
+            <tr>
+                <td>database-name</td>
+                <td>required</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>Database name of OceanBase to monitor.</td>
+            </tr>
+            <tr>
+                <td>table-name</td>
+                <td>required</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>Table name of OceanBase to monitor.</td>
+            </tr>
+            <tr>
+                <td>hostname</td>
+                <td>optional</td>
+                <td style="word-wrap: break-word;">(none)</td>
+                <td>String</td>
+                <td>IP address or hostname of the OceanBase database server or OceanBase Proxy server.</td>
+            </tr>
+            <tr>
+                <td>port</td>
+                <td>optional</td>
+                <td style="word-wrap: break-word;">(none))</td>
```

Comment:
Does the default port of OceanBase is 2881?

Reply:
2881 is the default port for directly connecting to a single server, and 2883 is the default port for connecting with obproxy for cluster.

Reply:
> 2881 is the default port for directly connecting to a single server, and 2883 is the default port for connecting with obproxy for cluster.

@whhe No default port here?

Reply:
> > 2881 is the default port for directly connecting to a single server, and 2883 is the default port for connecting with obproxy for cluster.
> 
> @whhe No default port here?

There are different default values according to the specific connecting mode, so I think it's better to ask user setting it himself for snapshot reading to make sure it is configured properly.